tests/test_daemon.cpp

```diff
@@ -1078,6 +1078,32 @@ INSTANTIATE_TEST_SUITE_P(DaemonRefuseRelease, RefuseBridging,
 INSTANTIATE_TEST_SUITE_P(DaemonRefuseSnapcraft, RefuseBridging, Values(std::make_tuple("snapcraft", "core")));
 INSTANTIATE_TEST_SUITE_P(DaemonRefuseRemoteless, RefuseBridging, Combine(Values(""), ValuesIn(old_remoteless_rels)));
 
+TEST_F(Daemon, fails_with_image_not_found_also_if_image_is_also_non_bridgeable)
+{
+    auto mock_image_host = std::make_unique<NiceMock<mpt::MockImageHost>>();
+    auto mock_image_host_ptr = mock_image_host.get();
+
+    std::vector<mp::VMImageHost*> hosts;
+    hosts.push_back(mock_image_host_ptr);
+
+    mp::URLDownloader downloader(std::chrono::milliseconds(1));
+    mp::DefaultVMImageVault default_vault(hosts, &downloader, mp::Path("/"), mp::Path("/"), mp::days(1));
+
+    auto mock_image_vault = std::make_unique<NiceMock<mpt::MockVMImageVault>>();
+    auto mock_image_vault_ptr = mock_image_vault.get();
+
+    EXPECT_CALL(*mock_image_vault_ptr, all_info_for(_)).Times(1).WillOnce([&default_vault](const mp::Query& query) {
```

Comment:
A single `WillOnce` implies `Times(1)`, so you can drop the `Times` here if you'd like, although it's not necessary.

Reply:
Oh, didn't know that. I removed `Times(1)` to make the code clearer. Thanks!